tests/Doctrine/Tests/Common/Annotations/LegacyCachedReaderTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Doctrine\Tests\Common\Annotations;
+
+use Doctrine\Common\Cache\Cache;
+use Doctrine\Common\Cache\CacheProvider;
+use Doctrine\Tests\Common\Annotations\Fixtures\Annotation\Route;
+use Doctrine\Common\Annotations\AnnotationReader;
+use Doctrine\Common\Annotations\CachedReader;
+use Doctrine\Common\Cache\ArrayCache;
+
+class LegacyCachedReaderTest extends AbstractReaderTest
```

Comment:
This is a copy of `CachedReaderTest` before this PR, so that the deprecated code path is still tested and can be easily removed in the next major release.